frontend/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,188 @@
+import { useController, useForm } from 'react-hook-form'
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+} from '@chakra-ui/form-control'
+import { CheckboxGroup, CheckboxProps, Input, VStack } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+import { omit } from 'lodash'
+
+import Button from '~components/Button'
+
+import { Checkbox } from './Checkbox'
+import { CheckboxOthers } from './CheckboxOthers'
+
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  decorators: [],
+} as Meta
+
+export const Default: Story<CheckboxProps> = (args) => <Checkbox {...args} />
+Default.args = {
+  value: 'Option',
+}
+
+export const Group: Story<CheckboxProps> = (args) => {
+  return (
+    <VStack align="left">
+      <Checkbox value="Option 1" />
+      <Checkbox value="Option 2" />
+      <Checkbox value="Option 3" />
+      <CheckboxOthers {...args} value="Others">
+        <Input placeholder="Please specify" />
+      </CheckboxOthers>
+    </VStack>
+  )
+}
+
+export const Playground: Story = (args) => {
+  const { name, label, isDisabled, isRequired } = args
+
+  const { handleSubmit, watch, control, register } = useForm()
+  const {
+    field,
+    formState: { errors },
+  } = useController({
+    control,
+    name,
+    rules: {
+      required: isRequired
+        ? { value: true, message: 'This field is required' }
+        : false,
+    },
+  })
+
+  const values = watch(name)
+
+  const onSubmit = (data: Record<string, string>) => {
+    alert(JSON.stringify(data))
+  }
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} noValidate>
+      <FormControl
+        isRequired={isRequired}
+        isDisabled={isDisabled}
+        isInvalid={!!errors[name]}
+        mb={6}
+      >
+        <FormLabel htmlFor={name}>{label}</FormLabel>
+        <VStack align="left">
+          <CheckboxGroup {...omit(field, 'ref')}>
```

Comment:
why is ref omitted here?

Reply:
`CheckboxGroup` doesn't have `forwardRef` so passing in a ref causes an error.

Reply:
Then where is the field's ref being passed into? This would be used to focus on the field when any errors occur. 

Maybe we don't even need to use a controller for this. We can just use `register` directly on Checkbox and Checkbox others